### PR TITLE
(feat) O3-3656: Change the style of abnormal and dangerous lab results

### PR DESCRIPTION
**Pull request by CynthiaKamau**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

- Updated the styling of Abnormal and Dangerous results to be orange and red respectively

## Screenshots

https://github.com/user-attachments/assets/6148e3a8-a77b-4bdb-9e0c-3b1fe9321533

## Related Issue

https://openmrs.atlassian.net/browse/O3-3656

### CODE REVIEW

File: packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss [hunk 1]
```diff
@@ -139,12 +140,12 @@
 
 .high,
 .low {
-  box-shadow: 0 0 0 1px #000000;
+  background-color: colors.$orange-20;
```
Comment (denniskigen) [inline packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss:143]: I think this should be `colors.$orange-10` per the hex code in Ciaran's screenshot https://openmrs.atlassian.net/browse/O3-3656.

Comment (denniskigen) [reply packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss:143]: Also, per the acceptance criteria in the ticket, you'd need to implement the hover state backgrounds.

Comment (denniskigen) [reply packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss:143]: I've sought clarification from Ciaran in the issue thread.

**Review by arodidev — COMMENTED** [review]
Thanks @CynthiaKamau, shouldn't we adopt the new results table design specified in [this design guide](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/66a35f369644e593803cf0f6)? Or would that be outside the scope of this ticket?

**Comment by denniskigen** [comment]
The background color on hover should be a blue hue:

b740071a-c7ac-40de-8d40-f3e8c3c27c6b

**Comment by CynthiaKamau** [comment]
> The background color on hover should be a blue hue:
> 
> b740071a-c7ac-40de-8d40-f3e8c3c27c6b
> 
> I've asked Ciaran to furnish us with the exact hex codes in the issue thread.

Done

**Comment by CynthiaKamau** [comment]
I was actually making more edits

File: packages/esm-patient-labs-app/src/test-results/panel-view/panel.component.tsx [hunk 2]
```diff
@@ -90,15 +90,12 @@ const LabSetPanel: React.FC<LabSetPanelProps> = ({ panel, observations, activePa
...
-          [styles.activePanel]: activePanel?.conceptUuid === panel.conceptUuid,
-        })}
-        onClick={() => setActivePanel(panel)}
-        role="button"
-        tabIndex={0}
-      >
+    <Layer
+      className={classNames(styles.labSetPanel, {
+        [styles.activePanel]: activePanel?.conceptUuid === panel.conceptUuid,
+      })}
```
Comment (denniskigen) [inline packages/esm-patient-labs-app/src/test-results/panel-view/panel.component.tsx:96]: Moving this styling to the parent container fixes another visual artifact:

CleanShot 2024-07-31 at 12  59 28@2x

File: packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss [hunk 3]
```diff
@@ -139,12 +144,14 @@
 
 .high,
 .low {
-  box-shadow: 0 0 0 1px #000000;
+  background-color: colors.$orange-10;
```
Comment (denniskigen) [inline packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss:147]: ### Before

CleanShot 2024-07-31 at 1  01 11@2x

### After

CleanShot 2024-07-31 at 1  02 02@2x

File: packages/esm-patient-labs-app/src/test-results/grouped-timeline/grouped-timeline.styles.scss [hunk 4]
```diff
@@ -64,6 +65,11 @@
     @include type.type-style('body-compact-01');
     color: $text-02;
   }
+
+  &:hover {
+    background-color: colors.$blue-10;
+    border-top: 2px colors.$blue-30;
+  }
 }
```
Comment (denniskigen) [inline packages/esm-patient-labs-app/src/test-results/grouped-timeline/grouped-timeline.styles.scss:73]: These changes effect the same styling mods made to the Timeline view to the GroupedTimeline view (what you see after selecting a panel in the tree view):

CleanShot 2024-07-31 at 1  03 01@2x

File: packages/esm-patient-labs-app/src/test-results/panel-view/result-panel.scss [hunk 5]
```diff
@@ -51,12 +52,18 @@
...
+  }
 }
 
 .critically-high,
 .critically-low {
-  border: 2px solid $danger !important;
+  td {
+    background-color: colors.$red-20;
+    border-top: 1px solid colors.$red-20-hover;
+  }
```
Comment (denniskigen) [inline packages/esm-patient-labs-app/src/test-results/panel-view/result-panel.scss:66]: ### Before

CleanShot 2024-07-31 at 1  05 34@2x

### After

CleanShot 2024-07-31 at 1  05 15@2x

**Review by denniskigen — COMMENTED** [review]
Sorry for blowing your PR up a touch, but I had to make some additional changes to:

- Fix the styling used in Panel view to highlight abnormal values.
- Extend the changes in the Timeline view to the GroupedTimeline view.
- Fix component names to match our [naming conventions](https://o3-docs.openmrs.org/docs/coding-conventions#naming) so that i18next-parser knows to extract translation strings and keys from them.

**Review by denniskigen — APPROVED** [review]

**Pull request merged**